CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ endif()
 
 option(MAGIC_ENUM_OPT_BUILD_EXAMPLES "Build magic_enum examples" ${IS_TOPLEVEL_PROJECT})
 option(MAGIC_ENUM_OPT_BUILD_TESTS "Build and perform magic_enum tests" ${IS_TOPLEVEL_PROJECT})
+option(MAGIC_ENUM_OPT_INSTALL "Generate install target" ON)
```

Comment:
Maybe  `Generate and install magic_enum target"`?  How do I know it installs, or only generates?